# HG changeset patch
# Parent 33c4b64acc59b01c01ab5fc4d146b7402d8002b7
Build: Disable several whole directories

On the chopping block include:
 -flask
 -examples
 -libaio
 -blktap
 -blktap2
 -remus
 -gdbsx

diff -r 33c4b64acc59 tools/Makefile
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -8,10 +8,8 @@ endif
 SUBDIRS-y :=
 SUBDIRS-y += include
 SUBDIRS-y += libxc
-SUBDIRS-y += flask
 SUBDIRS-y += xenstore
 SUBDIRS-y += misc
-SUBDIRS-y += examples
 SUBDIRS-y += hotplug
 SUBDIRS-y += xentrace
 SUBDIRS-$(CONFIG_XCUTILS) += xcutils
@@ -19,12 +17,7 @@ SUBDIRS-$(CONFIG_X86) += firmware
 SUBDIRS-y += console
 SUBDIRS-y += xenmon
 SUBDIRS-y += xenstat
-SUBDIRS-$(CONFIG_Linux) += $(SUBDIRS-libaio)
 SUBDIRS-$(CONFIG_Linux) += memshr 
-ifeq ($(CONFIG_X86),y)
-SUBDIRS-$(CONFIG_Linux) += blktap
-endif
-SUBDIRS-$(CONFIG_Linux) += blktap2
 SUBDIRS-$(CONFIG_NetBSD) += $(SUBDIRS-libaio)
 SUBDIRS-$(CONFIG_NetBSD) += blktap2
 SUBDIRS-$(CONFIG_NetBSD) += xenbackendd
@@ -40,9 +33,7 @@ endif
 
 SUBDIRS-y += xenpmd
 SUBDIRS-y += libxl
-SUBDIRS-y += remus
 SUBDIRS-$(CONFIG_X86) += xenpaging
-SUBDIRS-$(CONFIG_X86) += debugger/gdbsx
 SUBDIRS-$(CONFIG_X86) += debugger/kdd
 SUBDIRS-$(CONFIG_TESTS) += tests
 
